core/amber/src/main/scala/edu/uci/ics/amber/engine/common/ambermessage/WorkflowMessage.scala

```diff
@@ -21,13 +51,13 @@ case object WorkflowMessage {
 }
 
 case class WorkflowControlMessage(
-    from: ActorVirtualIdentity,
+    channel: ChannelEndpointID,
```

Comment:
now we lose the information whether it is `from` or `to`. better to make it clear.